examples/view-integration/external-views/package.json

```diff
@@ -39,7 +39,6 @@
 		"webpack:dev": "webpack --env development"
 	},
 	"dependencies": {
-		"@fluid-example/example-utils": "workspace:~",
 		"@fluid-internal/client-utils": "workspace:~",
```

Comment:
Should we also look to remove this dependency?

Reply:
In general I'd love to - currently it's only for TypedEventEmitter which I don't think we have a non-internal substitute for yet.  Could use an external library but I might defer that decision for now.